doc/integrations/AWS_EC2.md

```diff
@@ -0,0 +1,129 @@
+# Running CORTX on AWS
+
+
+Summary:
+----
+This document walks you through how to set up CORTX on an Amazon EC2 instance and how to perform IO operations using another EC2 instance on the same subnet.
+
+Prerequisites:
+-------
+
+- You will need an AWS account with right to create an [EC2 instances.](https://aws.amazon.com/ec2/?ec2-whats-new.sort-by=item.additionalFields.postDateTime&ec2-whats-new.sort-order=desc) 
+- You will need to [import to an AMI](https://docs.aws.amazon.com/vm-import/latest/userguide/vmimport-image-import.html) the [CORTX OVA.](https://github.com/Seagate/cortx/releases/download/VA/cortx-va-1.0.2.zip) 
```

Comment:
Hi @justinzw Can we publish AMI in AWS Marketplace.  So that users not required to import an AMI but can select it from Marketplace directly.

Reply:
Yes, we want to do that eventually but this is still a good intermediate step while we wait to get there.

Reply:
Also, Justin, how will we remember to update this page when the OVA release is updated?